### ISSUE TITLE
Subtasks [feature request]

### ISSUE THREAD

**Issue by senpl** [codaxy/tdo#12]
Currently all tasks are main tasks. It is common that tasks got subtasks which allow for fast outline of what is to be done to finish specific task. Especially during planning or collecting information.

I think it should look like:
[  ] main task
   [  ] subtask1
   [  ] subtask2

I know it could be obtained with use of hashtags and css's still simpler general solution would be nice for everyday users.

**Comment by senpl** [comment]
I think it could be implemented with https://github.com/frontend-collective/react-sortable-tree to gives user more sorting options, but it is just one of many options.

**Comment by mstijak** [comment]
I agree that this feature would be very cool, but requires a lot of work.
Tasks should get a field indicating the parent task. 
CxJS supports drag & drop, although it is not yet implemented in tdo (#8).
https://codesandbox.io/s/2povoy3m8r

**Comment by senpl** [comment]
I implement it in very simple way. Still I would like to know which keyboard shortcut use for making task subtask and making subtask main task. My sugestion is "s" key for making subtask and "m" key for making task from subtask to main task. Of course much more options with subtask could be implemented(like expands and callapse), still to decide which one will really be needed.

**Comment by senpl** [comment]
I created first rough version in https://github.com/senpl/tdo . Now I want to have expands and callapse. Any recommended lib or solution to use ??? I think about https://github.com/sk-rt/handy-collapse but at this point anything could work.
